src/docs/sphinx/basicExamples/gravityInducedStressInitialization/Example.rst

```diff
@@ -0,0 +1,232 @@
+.. _gravityinducedhydrostaticinitialization:
+
+
+#################################################################################
+ Model Initialization: Hydrostatic and Mechanical Equilibrium
```

Comment:
Maybe instead of going under Basic Examples this needs its own section about Poromechanical models initialization.

Reply:
Yes, create a dedicated section about initialization in the User Guide